Common/Models/StatusExtensionContext.swift

```diff
@@ -53,6 +53,52 @@ struct PredictedGlucoseContext {
     }
 }
 
+struct DeviceStatusHighlightContext: DeviceStatusHighlight {
+    var localizedMessage: String
+    var imageSystemName: String
```

Comment:
I'm _guessing_ it means the `systemName` of the `Image` to use for the highlight (i.e. https://developer.apple.com/documentation/swiftui/image/init(systemname:) )

Reply:
@rickpasetto That is correct. 

This will be updated to just `imageName` as some of the these images are system and others are coming from the asset catalog.